Gemfile

```diff
@@ -47,7 +47,7 @@ gem "listen",                           "~>3.2",             :require => false
 gem "log_decorator",                    "~>0.1",             :require => false
 gem "manageiq-api-client",              "~>0.3.4",           :require => false
 gem "manageiq-loggers",                 "~>0.6.0",           :require => false
-gem "manageiq-messaging",               "~>1.0",             :require => false
+gem "manageiq-messaging",               "~>1.0", ">=1.0.3",  :require => false
```

Comment:
In hindsight, we should have cut a 1.1, since it introduced a new, backward compatible, feature.